docroot/profiles/custom/cgov_site/modules/custom/pdq_core/pdq_core.install

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/**
+ * @file
+ * Install pdq_core module.
+ */
+
+use Drupal\cgov_core\CgovCoreTools;
+use Drupal\user\Entity\User;
+
+/**
+ * Implements hook_install().
+ *
+ * Create the account used for importing PDQ content.
+ */
+function pdq_core_install() {
+
+  // Get our helper.
+  $siteHelper = \Drupal::service('cgov_core.tools');
+
+  // Add content type permissions and assign to workflow.
+  $restfulPermissions = [
+    'restful delete pdq_api',
+    'restful get pdq_api',
+    'restful post pdq_api',
+  ];
+  $siteHelper->addContentTypePermissions('pdq_cancer_information_summary', ['pdq_importer'], CgovCoreTools::DEFAULT_PERMISSIONS);
+  $siteHelper->addRolePermissions(['pdq_importer' => $restfulPermissions]);
+  $user = User::create();
+  $user->setUsername('PDQ');
+  $user->setEmail('pdq@cancer.gov');
+  $user->enforceIsNew();
+  $user->setPassword(getenv('PDQ_IMPORTER_PASS'));
```

Comment:
This is OK for now, but we should add a backlog item for setting up API keys. That way we can rotate them via a env var and not a password that is set in the DB.

Reply:
I've entered a draft of how I understand this as #594.